MinimedKit/PumpEvents/PlaceholderPumpEvent.swift

```diff
@@ -9,26 +9,33 @@
 import Foundation
 
 
-public struct UnknownPumpEvent: TimestampedPumpEvent {
+public struct PlaceholderPumpEvent: TimestampedPumpEvent {
     public let length: Int
     public let rawData: NSData
     public let timestamp: NSDateComponents
+    public let name: String
```

Comment:
Rather than storing the string in the struct (which requires heap allocation), let's just move the name generation code to the `dictionaryRepresentation` impl.